akka-discovery/src/main/resources/reference.conf

```diff
@@ -0,0 +1,75 @@
+######################################################
+# Akka Discovery Config                              #
+######################################################
+
+akka.actor.deployment {
+  "/SD-DNS/async-dns" {
+    mailbox = "unbounded"
+    router = "round-robin-pool"
+    nr-of-instances = 1
+  }
+}
+
+akka.discovery {
+
+  # Users MUST configure this value to set the default discovery method.
```

Comment:
```suggestion
  # Users MUST configure this value to set the default discovery mechanism.
```

Reply:
we voted and decided to use `method` every where and remove `mechanism`

Reply:
👍 k